### ISSUE TITLE
Browserify not getting the browser build specifiecd in package.json

### ISSUE THREAD

**Issue by danielstocks** [rtsao/styletron#208]
After upgrading to Styletron 3.x we we're experiencing Browserify build errors related to Styletron.

## Background

I'm looking at https://github.com/rtsao/styletron/blob/master/packages/styletron-react/package.json and it has the following lines: 
```
{
  "main": "./dist/index",
  "module": "./dist/index.mjs",
  "browser": {
    "./dist/index.js": "./dist/browser.es5.js",
    "./dist/index.mjs": "./dist/browser.es5.mjs"
},
```

The reason our build is failing is because we're piping our browserify bundle through uglify for JS minification. But now it won't run anymore because styletron 3.x is giving us non ES5 compatible js when importing it. 

**Styletron 2.x works fine** in our build pipeline becuase importing it give's us ES5/Browser compatible code, that will run through uglify without any errors. 

## Example code:

```js
const b = browserify()
b.require('styletron-react')
b.bundle().pipe(process.stdout);
```

## Result:

Gives me the contents of `node_modules/styletron-react/dist/index.js`

## Expected:

If i understood Browserify correctly, should I not get `node_modules/styletron-react/dist/browser.es5.js` ?

**Comment by rtsao** [comment]
It appears browserify and webpack differ in how they treat the `browser` field, which means it isn't working with browserify. Working on a fix for this.

**Comment by rtsao** [comment]
This should be resolved in `3.0.2`

**Comment by danielstocks** [comment]
I can confirm 3.0.3 solves the issue. Thank you.
